pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -205,8 +208,8 @@ func (a *Bootstrap) addStorageFiles(base string, uri string, templateData *boots
 	}
 	ign := ignition.FileFromBytes(strings.TrimSuffix(base, ".template"), mode, data)
 	if filename == ".bash_history" {
-		ign.User = &igntypes.NodeUser{Name: "core"}
-		ign.Group = &igntypes.NodeGroup{Name: "core"}
+		ign.User = &igntypes.NodeUser{Name: ignitionUser}
+		ign.Group = &igntypes.NodeGroup{Name: ignitionUser}
```

Comment:
Overloading, mumble, mumble.  But I'm fine adding an `ignitionGroup` later if we end up needing it.